editor/treemodel.dre

```diff
@@ -311,7 +295,7 @@ either express or implied. See the License for the specific language governing p
                   title:name, 
                   image:config.img_root + (tagsWithIcons[tag] ? tag : 'view') + '.png', 
                   id:id,
-                  ref:view.isRoot() && view.isA(dr.View) ? editor.content_bg.content.inner : view
+                  __ref:view.isRoot() && view.isA(dr.View) ? editor.content_bg.content.inner : view
```

Comment:
Note, the tree ignores any attributes with names that start with '__', so in order to keep the underlying Dreem object that's attached to the tree data from also being walked by the tree-diff (which is very slow and not necessary), I renamed it to '__ref'.